qiita_db/study.py

```diff
@@ -206,6 +207,26 @@ def get_by_status(cls, status):
 
         return studies
 
+    @staticmethod
+    def get_by_portal(portal):
```

Comment:
I can remove the function then, but once removed the pull request is good to go.